script/install_clang.sh

```diff
@@ -36,8 +36,35 @@ then
     sudo add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-13 main'
 fi
 
+# add clang-13 repository for ubuntu 22.04
+if [[ "$(cat /etc/os-release)" == *"22.04"* && "${ALPAKA_CI_CLANG_VER}" -eq 13 ]]
+then
+    wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+    sudo add-apt-repository 'deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-13 main'
+fi
+
+# add clang-14 repository for ubuntu 20.04
+if [[ "$(cat /etc/os-release)" == *"20.04"* && "${ALPAKA_CI_CLANG_VER}" -eq 14 ]]
+then
+    wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+    sudo add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-14 main'
+fi
+
+# add clang-14 repository for ubuntu 22.04
+if [[ "$(cat /etc/os-release)" == *"22.04"* && "${ALPAKA_CI_CLANG_VER}" -eq 14 ]]
+then
+    wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+    sudo add-apt-repository 'deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-14 main'
+fi
+
```

Comment:
I am by no means skilled in bash. Could we have something like that:
```suggestion
codename=`lsb_release -c  | awk '{print $2}'`
wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
sudo add-apt-repository "deb http://apt.llvm.org/$codename/ llvm-toolchain-$codename-$ALPAKA_CI_CLANG_VER main"
```

Reply:
Thanks for the suggestion, I picked this up in #1763.